### PR TITLE
Update photos app

### PR DESCRIPTION
**Pull request by spencerkimball**
- Update DB URL to port 26257 and specify the database name
- Don't use `photos.` prefix in SQL code (assume database was specified in conn params)
- Make use of efficient limit queries for finding an existing user id (and use order by)

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/examples-go/42)

### CODE REVIEW

File: photos/db.go [hunk 1]
```diff
@@ -148,26 +149,15 @@ SELECT id FROM photos.users WHERE id = $1`
...
-				return 0, err
-			}
-	*/
-	exists, err := userExists(tx, userID)
-	if err != nil {
-		return 0, err
-	} else if !exists {
+	var id int
+	const selectSQL = `
+SELECT id FROM users WHERE id >= $1 ORDER BY id LIMIT 1;
```
Comment (tamird) [inline photos/db.go]: isn't this just `SELECT MIN(id) FROM users WEHRE id >= $1`?

Comment (tamird) [inline photos/db.go]: Interestingly, this sort of query can confuse MySQL https://news.ycombinator.com/item?id=9983374

Comment (spencerkimball) [inline photos/db.go]: Sure, they're equivalent.

File: photos/db.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ const (
 	photosSchema = `
 CREATE DATABASE IF NOT EXISTS photos;
```
Comment (petermattis) [inline photos/db.go]: Sort of strange that you're creating the `photos` database here while also requiring it in the URL. Perhaps you should pull out the database out of the supplied URL and ensure that database exists.

**Comment by bdarnell** [comment]
LGTM

File: photos/db.go (hunk 2 above)
Comment (spencerkimball) [inline photos/db.go]: Ah, I see what you're saying. Hmm, I'll add a TODO.

**Pull request merged**